Update active-directory-b2c-troubleshoot-custom.md

marcelodiiorio opened the pull request:
Must be userJourney, instead of policyName. I forgot to propose the update of the sample code.

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

marcelodiiorio committed:
Update active-directory-b2c-troubleshoot-custom.md

Must be userJourney, instead of policyName. I forgot to propose the update of the sample code.

GitHubber17 committed:
Update active-directory-b2c-troubleshoot-custom.md
